### DISCUSSION TITLE
DirectML and FP8?

### DISCUSSION THREAD

**Discussion by dm3rd** [lshqqytiger/stable-diffusion-webui-directml#424] [Q&A] [answered]
I've tried looking this up but was unable to find anything.

All the posts seem to point that FP8 uses much less memory than FP16/32. as I am stuck with a rx470 4gb, I was hoping to use it to get better sized images, even if it is only 100px or so.

the cmd window says I am using torch 2.0.0+cpu, I tried updating with --reinstall-torch.
On it's own It updated but throws an error and I can't use a1111, if I do it with --use-directml it won't update and uses the same as above.

Am I missing something or are they incompatable?

**Comment by lshqqytiger** [comment, marked as answer]
Incompatible.
Actually, it seems that Microsoft has dropped torch-directml project. The latest version of torch that is compatible with torch-directml is 2.0.0, and you can't upgrade it unless they release a newer version.

**Reply by dm3rd** [reply]
Well that sucks major balls, thanks for the reply anyway.
